core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperatorTableFactory.java

```diff
@@ -117,8 +117,7 @@ private SqlOperatorTable create(ImmutableSet<SqlLibrary> librarySet) {
     SqlOperatorTable operatorTable = SqlOperatorTables.of(list);
     if (standard) {
       operatorTable =
-          SqlOperatorTables.chain(SqlStdOperatorTable.instance(),
-              operatorTable);
+          SqlOperatorTables.chain(operatorTable, SqlStdOperatorTable.instance());
```

Comment:
This may cause problems for other people. Ideally you want to do this just for your tests.

Reply:
Understood. I'll take a look.

Reply:
This has been fixed. The only changes needed now are in the unit test itself. I setup the operator table there such that PostgreSQL is first.